bindings/http/http.go

```diff
@@ -104,6 +107,13 @@ func (h *HTTPSource) Init(metadata bindings.Metadata) error {
 		h.errorIfNot2XX = true
 	}
 
+	if val, ok := metadata.Properties["traceHeaders"]; ok {
+		h.traceHeaders = utils.IsTruthy(val)
+	} else {
+		// Default behavior
+		h.traceHeaders = false
+	}
```

Comment:
We do not need the else block here. default will be false only IMO.

Reply:
Done